### PR TITLE
Mothroach DNA infusion

### PR DESCRIPTION
**Pull request by GuillaumePrata**
## About The Pull Request
It has 4 organ options, all default ones from Mothpeople.
Moth eyes: Weak to flash
Moth tongue: Makes you flutter when you talk
Moth antennae: Makes you weaker to noogies
Moth wings: Just normal moth wings

Also ABC'd some lists while there.
## Why It's Good For The Game
I love the infuser and want more content to it, even if this one is mostly memery...
A fun side of the DNA infuser is making a mish mash of species organs into someone, the moth organs are mostly downsides but people still like how they look, win-win.
## Changelog
:cl: Guillaume Prata
add: Mothroach DNA infusion. For better or worse, anyone can get closer to being a Moth person now!
/:cl:

### CODE REVIEW

**Comment by Autisem** [comment]
give them stomach/tongue that lets them eat garbage for there roach side

File: code/game/machinery/dna_infuser/infuser_entries.dm [hunk 1]
```diff
@@ -151,9 +151,31 @@ GLOBAL_LIST_INIT(infuser_entries, prepare_entries())
...
 		/obj/item/organ/internal/heart/goliath,
-		/obj/item/organ/internal/brain/goliath,
+		/obj/item/organ/internal/lungs/lavaland/goliath,
 	)
 	infusion_desc = "armored tendril-like"
+
+/datum/infuser_entry/mothroach
+	name = "Mothroach"
+	infuse_mob_name = "Mothroach"
+	desc = "So first they mixed moth and roach dna to make mothroaches, and now we mix mothroach dna with humanoids to make mothmen hybrid?"
```
Comment (Zonespace27) [inline code/game/machinery/dna_infuser/infuser_entries.dm:164]:
```suggestion
	desc = "So first they mixed moth and roach DNA to make mothroaches, and now we mix mothroach DNA with humanoids to make mothmen hybrids?"
```

**Comment by Zytolg** [comment]
This is against god and nature

**Comment by Serxule** [comment]
mothroaches are kinda very rare though (unless you have enough money for 700 per through cargo)

**Comment by Bluedino1025** [comment]
> mothroaches are kinda very rare though (unless you have enough money for 700 per through cargo)

Science can order them with their department console and seeing as most other science things rely on toxins (which no one does half the time) it’s fairly possible

File: code/game/machinery/dna_infuser/infuser_entries.dm [hunk 2]
```diff
@@ -151,9 +151,31 @@ GLOBAL_LIST_INIT(infuser_entries, prepare_entries())
...
+/datum/infuser_entry/mothroach
+	name = "Mothroach"
+	infuse_mob_name = "Mothroach"
+	desc = "So first they mixed moth and roach DNA to make mothroaches, and now we mix mothroach DNA with humanoids to make mothmen hybrids?"
+	threshold_desc = NO_THRESHOLD
+	qualities = list(
+		"eyes weak to bright lights",
+		"you flutter when you talk",
+		"wings that can't even carry your body weight",
+		"i hope it was worth it",
```
Comment (Time-Green) [inline code/game/machinery/dna_infuser/infuser_entries.dm:170]: eyes weak, body heavy, moth spaghetti

**Review by Time-Green — APPROVED** [review]

**Pull request merged**